packages/web/src/flash/Flash.tsx

```diff
@@ -1,66 +1,10 @@
-import { useEffect, useState } from 'react'
+import { Toaster } from 'react-hot-toast'
 
-import { useFlash } from 'src/flash/FlashContext'
-import type { FlashMessage } from 'src/flash/FlashReducer'
-
-type FlashMessageProps = {
-  message: FlashMessage
-  timeout?: number
-}
-
-const FlashMessageComponent = ({ message, timeout }: FlashMessageProps) => {
-  const { dismissMessage, cycleMessage } = useFlash()
-  const [classes, setClasses] = useState('')
-
-  useEffect(() => {
-    cycleMessage(message.id)
-    // cycleMessage should not trigger update
-  }, [cycleMessage, message.id])
-
-  useEffect(() => {
-    if (timeout === undefined) {
-      return
-    }
-
-    const fadeOutTimer = setTimeout(() => {
-      setClasses('rw-slide-up')
-    }, timeout)
-
-    return () => clearTimeout(fadeOutTimer)
-  }, [timeout])
-
-  return (
-    <div
-      className={`rw-flash-message ${message.classes} ${classes}`}
-      style={message.style}
-      data-testid="message"
-    >
-      <div className="rw-flash-message-text">{message.text}</div>
-      <div
-        className="rw-flash-message-dismiss"
-        data-testid="dismiss"
-        onClick={() => dismissMessage(message.id)}
-      >
-        +
-      </div>
-    </div>
-  )
-}
-
-const Flash = ({ timeout }: { timeout?: number }) => {
-  const { messages } = useFlash()
-
-  if (!messages.length) {
-    return null
-  }
-
-  return (
-    <div className="rw-flash" data-testid="flash">
-      {messages.map((msg) => (
-        <FlashMessageComponent key={msg.id} message={msg} timeout={timeout} />
-      ))}
-    </div>
+const Flash = () => {
```

Comment:
You can also use to get design-time warnings in VSCode. It'll be struck through.

`/** @deprecated this thing is going to go away. */`

Reply:
Ooo okay I’ll add that

Reply:
Looks like that only deprecates it in the Redwood package itself, not if you use it in your app? Or am I doing it wrong?

Reply:
I think it's correct, I'll try to see why it isn't working outside the project.

Reply:
Should we still accept an optional `timeout`, to keep backwards compatibility with ppl using `<Flash timeout={3000} />` in TS projects

Reply:
Hmm we could, we'd have to forward that value to the more in-depth version that Toast uses, where you set the timeout per type (success, error, loading...): https://react-hot-toast.com/docs/toaster  Although we're already providing a big ol' deprecation notice which should make it pretty clear why your toast may not be showing for the same amount of time it used to (in addition to looking completely different).

Question: how come you called out TS projects here specifically? Is there some different usage from plain JS?

Reply:
Yes, with the code as it is, in a TS project you'd ge a red squiggly line, saying that <Flash> doesn't accept a `timeout` prop. JS doesn't care if you pass extra stuff, but TS does.

So I'm just suggesting to add it, to make the code compile. Not implement the actual functionality